vcbuild.bat

```diff
@@ -332,7 +332,7 @@ copy /Y ..\README.md node-v%FULLVERSION%-win-%target_arch%\ > nul
 if errorlevel 1 echo Cannot copy README.md && goto package_error
 copy /Y ..\CHANGELOG.md node-v%FULLVERSION%-win-%target_arch%\ > nul
 if errorlevel 1 echo Cannot copy CHANGELOG.md && goto package_error
-robocopy /e ..\deps\npm node-v%FULLVERSION%-win-%target_arch%\node_modules\npm > nul
+robocopy ..\deps\npm node-v%FULLVERSION%-win-%target_arch%\node_modules\npm /e /xd test > nul
```

Comment:
[robocopy documentation](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) has the syntax as `robocopy <Source> <Destination> [<File>[ ...]] [<Options>]` so I've moved the existing option (`/e`) to the end of the command line in addition to the new option to exclude `test` directories.